components/bloks/WorkableList.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="vertical xs-full s-half h-padding-right clear-left">
+    <div v-if="loaded">
+      <workable-detail
+        :isOpen="detailOpen"
+        :shortcode="selectedShortcode"
+        :title="selectedTitle"
+        @close="onClose"
+        @closecomplete="onCloseComplete"
+      />
+      <div
+        v-for="(item, key) in availableJobsByCity"
+        :key="key"
+        class="v-margin-top border-both"
+      >
+        <h3>{{ key }}</h3>
```

Comment:
Hello @CharcoalStyles. For some reason, my change hasn't made it into this PR. I changed the H3 to a H2.